.github/workflows/record_snapshots.yml

```diff
@@ -155,7 +156,7 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: `### '${{ matrix.variant }}' snapshots were updated. Please review it.
+              body: `'${{ matrix.variant }}' snapshots were updated. Please review it.
```

Comment:
Why not leave this as heading.3 (###)?

Reply:
It'll be 4 comments with large headings. I think it's rather distracting.

Reply:
This creates an inconsistency with
> 'dm' files were changed. 

type of comments. 

Do you plan to ensure the format is the same across both?

Reply:
The comments you're mentioning are warning comments. It signalizes that something went wrong and developer attention is required.

These types of comments signalize that the task ran successfully.